.github/workflows/publish-npm.yml

```diff
@@ -1,12 +1,14 @@
 name: npmjs
+
 on:
-  release:
-    types: [published]
-  workflow_dispatch:
+  push:
+    branches:
+      - main
 
 jobs:
   publish:
     runs-on: ubuntu-latest
+    if: github.repository == 'ngrok-oss/mantle'
```

Comment:
I think technically we should have this guard, since we're a public repo that could get forked.